services/database/scripts/remove2023QualifierHPCAD.ts

```diff
@@ -0,0 +1,149 @@
+import * as readline from "readline";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  paginateScan,
+  UpdateCommand,
+} from "@aws-sdk/lib-dynamodb";
+
+/** Possibly overwritten by user input */
+let STAGE_NAME = "local";
+
+// As recommended by https://stackoverflow.com/a/68504470
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+const promptString = (query: string) =>
+  new Promise<string>((resolve) => rl.question(query, resolve));
+const promptYesNo = async (query: string) => {
+  do {
+    const userInput = await promptString(query);
+    switch (userInput.toUpperCase()[0]) {
+      case "Y":
+        return true;
+      case "N":
+        return false;
+      default:
+        rl.write("Y or N only, please.\n");
+    }
+  } while (true);
+};
+
+async function remove2023QualifierHPCAD() {
+  const isLocal = await promptYesNo(
+    "Do you want to run this script locally? Y/N: "
+  );
+  if (!isLocal) {
+    STAGE_NAME = await promptString(
+      "What environment are we running on (e.g. master, val, production)? "
```

Comment:
```suggestion
      "What environment are we running on (e.g. master, val, prod)? "
```

Let's match the permanent QMR stage names precisely so no one gets confused.